src/lambda_streams/Numbers.java

```diff
@@ -11,81 +11,248 @@ public static void main(String[] args) {
+                return false;
+            }
+        };
+        LambdaOddValue.Pick(3);
 
-        OR
+        PickNumber LambdaEvenValue = (int i) -> {
+            if(nums.get(i) % 2 == 1){
+                System.out.println("This is false");
+                return true;
+            }
+            else {
+                System.out.println("This is true");
+                return false;
+            }
+        };
+        LambdaEvenValue.Pick(3);
 
-        lambdaOdd = () -> {};
-        isOdd(lambdaOdd);
-        etc...
+        PickNumber LambdaPrimeValue = (int i) -> {
+            for (int x = 2; x < Math.sqrt(i); x++) {
+                if (i % x == 0) {
+                    System.out.println("False");
+                    return false;
+                }
+            }
+            System.out.println("True");
+            return true;
+        };
+        LambdaPrimeValue.Pick(3);
 
-        */
+        Calulate LambdaMaxValue = (nums) -> {
+            int max = 0;
+            for(int i = 0; i < nums.size(); i++){
+                if(nums.get(i) > max){
+                    max = nums.get(i);
+                }
+            }
+            System.out.println(max);
+            return max;
+        };
+        LambdaMaxValue.Compute(nums);
 
+        Calulate LambdaMinValue = (nums) -> {
+            int min = 0;
+            for(int i = 0; i < nums.size(); i++){
+                if(nums.get(i) < min){
+                    min = nums.get(i);
+                }
+            }
+            System.out.println(min);
+            return min;
+        };
+        LambdaMinValue.Compute(nums);
+
+        Comp LambdaComp = (int i, int j) -> {
+            if(nums.get(i) > nums.get(j)){
+                System.out.println(1);
+                return 1;
+            }
+            else if(nums.get(j) > nums.get(i)){
+                System.out.println(-1);
+                return -1;
+            }
+            else {
+                System.out.println(0);
+                return 0;
+            }
+        };
+        LambdaComp.Comparing(1,3);
+
+        Work LambdaAppend = (int n) -> {
+            nums.add(n);
+            System.out.println(n);
+            return n;
+        };
+        LambdaAppend.AddingIn(3);
+    }
+    interface Work{
+        public int AddingIn(int n);
+    }
+    interface Comp{
+        public int Comparing(int i, int j);
+    }
+    interface Calulate2{
+        public long Computer(List<Integer> nums);
     }
+    interface Calulate{
+        public int Compute(List<Integer> nums);
+    }
+    interface PickNumber {
+        public boolean Pick(int i);
+    }
+    public void sayHello(){
+        System.out.println("Say Hello");
+    }
+    interface added2{
+        public void someMethod();
 
+    }
     static boolean isOdd(int i) {
-        //determine if the value at the index i is odd.  return true if yes, return false if  no.
-        return false;
+        if(nums.get(i) % 2 == 1){
```

Comment:
What led to the decision to handle this in an 'If/Else' statement, rather than in one line (e.g. `return nums.get(i) % 2 == 1;` )?